Kmeans_python/silhouette.py

```diff
@@ -25,4 +81,18 @@ def silhouette(X, k_array):
     
     """
 
-    pass
+    scores = []
+    for i in range(len(k_array)):
+        centers, labels = fit(X, k_array[i])
+        score = sil_score(X, labels)
+        scores.append([k_array[i], score])
```

Comment:
Tested locally, and realized this block iterates from 0 to len(k_array), instead of over the actual k values. I think the following change should fix this

```suggestion
    for i, k in enumerate(k_array):
        centers, labels = fit(X, k)
        score = sil_score(X, labels)
        scores.append([k, score])
```

Using enumerate, we can still use i as index for loop iteration   
      

```

Reply:
Is there any functional difference between these two blocks? I think both should work, but if your implementation is better we can use it.

Reply:
I guess there is no functional difference, you're right. So makes no difference either way. Im happy to leave it as is